CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.55.1
```

Comment:
My understanding is that the CHANGELOG need to be added to main dart-sass repo and it will be synced to here by CI when we cut a new release.